src/App.tsx

```diff
@@ -1,5 +1,19 @@
+import colorData from '../public/colors.json';
+
+type Color = {
+    hex: string;
+    name: string;
+    comp: { hex: string; name: string }[];
+};
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    return colorData.map((color: Color) => {
+        return (
+            <div key={color.hex}>
```

Comment:
This is extremely limited. You're not using the complementary colors in any way or using the color itself to show something in the UI.